streamwebs_frontend/streamwebs/models.py

```diff
@@ -41,7 +41,7 @@ class Site(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     modified = models.DateTimeField(auto_now=True)
 
-    objects = SiteManager()
+    test_objects = SiteManager()
```

Comment:
@alxngyn Acc. to @athai, we won't need GeoManager and we should rename all the rest of the model's current ``objects`` to ``test_objects``. She says she will do it.